### ISSUE TITLE
"user.status" returns "online" for "idle" users

### ISSUE THREAD

**Issue by Jessassin** [hydrabolt/discord.js#197]
As title says. 

Given a bot that is idle, this will return "Bot status: online"

```
bot.sendMessage(msg.channel, "Bot status: " + bot.user.status)
```

**Comment by abalabahaha** [comment]
Is this for the bot itself, or for all users? It seems to be working fine for me when looking at other users.

**Comment by Jessassin** [comment]
As investigated in the chat, this appears to affect the bot only.

I have installed indev, and confirmed it currently still occurs.

**Comment by abalabahaha** [comment]
Might be somewhat fixed in 5c9865f
